doc/user/sql-grammar/keywords.txt

```diff
@@ -0,0 +1 @@
+../../../src/sql-parser/src/keywords.txt
```

Comment:
This is, I believe, the first doc that will be generated from source 🎉 

While I really don't think it matters much in this case, there is potential for this to reflect unstable builds and not the latest "blessed" release. Really small potatoes, but can I get you to make note of that in `doc/user/content/sql/identifiers.md`?

Reply:
Ah right, good point! Done.